src/main.cpp

```diff
@@ -23,27 +28,47 @@ uint8_t meanSampledBm = 0;
 void sampleAd8318()
 {
   uint32_t startSampleingTime = millis();
+  uint32_t adcIn = 0;
 
-  uint32_t tempReading = 0;
-  for (int i = 0; i < SCREEN_WIDTH; i++)
-  {
-    tempReading = analogRead(AD8318_PIN);
-    tempReading += analogRead(AD8318_PIN);
-    tempReading += analogRead(AD8318_PIN);
-    samples[i] = tempReading / 3;
-  }
+  #ifdef ARDUINO_TARGET
+    for (int i = 0; i < SCREEN_WIDTH; i++)
```

Comment:
Hmmm dat i++

Reply:
Missed one.... was trying to keep your existing code the same ;)